sqlite/src/test/java/org/jdbi/v3/sqlite3/TestUrls.java

```diff
@@ -45,15 +46,36 @@ public void tearDown() throws Exception {
 
     @Test
     public void testInsertUrlSuccessful() throws MalformedURLException {
-        String goolgeString = "http://www.google.com";
-        URL googleUrl = new URL(goolgeString);
+        String googleString = "http://www.google.com";
+        URL googleUrl = new URL(googleString);
 
         handle.createUpdate("INSERT INTO foo VALUES (:url)")
                 .bind("url", googleUrl)
                 .execute();
 
-        URL actualUrl = handle.createQuery("SELECT * FROM foo").mapTo(URL.class).findOnly();
+        URL actualUrl = handle.createQuery("SELECT url FROM foo").mapTo(URL.class).findOnly();
         Assertions.assertThat(actualUrl).hasToString(googleUrl.toString());
     }
 
+    @Test(expected = ResultSetException.class)
+    public void testMapNullUrlThrowsException() {
+        handle.createUpdate("INSERT INTO foo VALUES (:url)")
+                .bind("url", ((URL) null))
```

Comment:
The null case must be tested with `bindByType`. This case won't actually cause the URL argument to be used.